File: src/344.reverse-string.cs
public class Solution {
    public void ReverseString(char[] s)
    {
        int m = s.Length;
        if (m < 2)
            return;
        int ptr = 0;
        while (ptr < m - 1 - ptr)
        {
            char tmp = s[ptr];
            s[ptr] = s[m - 1 - ptr];
            s[m - 1 - ptr] = tmp;
            ptr++;
        }
    }
}

File: dump-cn/344.reverse-string.cpp
class Solution {
public:
	void reverseString(vector<char>& s) {
		int m = s.size();
		int ptr = 0;
		while (ptr < m - ptr - 1) {
			char tmp = s[ptr];
			s[ptr] = s[m - ptr - 1];
			s[m - ptr - 1] = tmp;
            ptr++;
		}
	}
};